src/main/java/com/amit/electronic/store/entity/User.java

```diff
@@ -9,24 +11,27 @@
 @Getter
 @Setter
 @NoArgsConstructor
+@Builder
 @AllArgsConstructor
 @ToString
 @Entity
 @Table(name = "users")
 public class User {
 
     @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
     @Column(name = "user_id")
-    public String id;
-    @Column(name = "user_name", length = 10)
-    public String name;
-    @Column(name = "user_email", unique = true, length = 15)
-    public String email;
-    @Column(name = "user_gender")
-    public String gender;
-    @Column(name = "user_password", length = 12)
-    public String password;
+    @JsonInclude
+    private Long id;
+    @Column(name = "user_name", length = 50)
+    private String userName;
+    @Column(name = "user_email", unique = true, length = 100)
+    private String userEmail;
+    @Column(name = "user_gender", length = 10)
+    private String userGender;
+    @Column(name = "user_password", length = 100)
+    private String userPassword;
     @Column(name = "user_description", length = 1000)
-    public String about;
+    private String userDescription;
```

Comment:
All of these field names do not need to be prefixed by user. Since these attributes are in the User class, it is expected that these attributes correspond to the User entity.

Reply:
done